mixer/adapter/stackdriver/log/log.go

```diff
@@ -106,14 +119,38 @@ func (b *builder) Build(ctx context.Context, env adapter.Env) (adapter.Handler,
 			_ = logger.Errorf("failed to evaluate template for log %s, skipping: %v", name, err)
 			continue
 		}
+
+		if log.SinkInfo != nil {
+			// first try to CreateSink, if error is AlreadyExists then update that sink.
+			_, err = syncClient.CreateSink(context.Background(), &logadmin.Sink{
```

Comment:
CreateSink returns sink and was not needed anywhere later..
I don't think we can avoid check for create v update unless we know that config for sink won't change, because say filter property of the sink changed and that sink already exists then we need to call update in that case.. 
what do you think?